language/google/cloud/language/document.py

```diff
@@ -266,10 +266,10 @@ def annotate_text(self, include_syntax=True, include_entities=True,
             This API is intended for users who are familiar with machine
             learning and need in-depth text features to build upon.
 
-        .. _annotateText: https://cloud.google.com/natural-language/\
+        .. _annotate-text: https://cloud.google.com/natural-language/\
```

Comment:
@jonparrott This introduced an error where `sphinx-docstring-typing` lowercased the target:

> Unknown target name: "annotatetext".